packages/next/src/trace/trace-uploader.ts

```diff
@@ -58,9 +59,17 @@ interface TraceMetadata {
   pkgName: string
   platform: string
   isTurboSession: boolean
+  nextVersion: string
 }
 
 ;(async function upload() {
+  const nextVersion = JSON.parse(
```

Comment:
curious if this'll work if next.js is installed as dep in normal project?

Reply:
Yeah, tested this and it works.